runtime/scripts/math.js

```diff
@@ -506,6 +506,26 @@ var math = Numbas.math = /** @lends Numbas.math */ {
         return Math.abs(a-b) <= Math.max( rel_tol * Math.max(Math.abs(a), Math.abs(b)), abs_tol );
     },
 
+    is_scalar_multiple(a,b,rel_tol,abs_tol) {
+      rel_tol = rel_tol === undefined ? 1e-15 : rel_tol;
+      abs_tol = abs_tol === undefined ? 1e-15 : abs_tol;
+
+      let ratios = [];
+
+      a.forEach(function (aElement, index) {
+          const bElement = b[index];
+          if (is_close(aElement, 0) || is_close(bElement, 0)) {
+              if (!is_close(aElement, bElement)) {
+                  return false;
+              }
+          } else {
+              ratios.push(aElement / bElement);
+          }
+          console.log(aElement, bElement);
+      })
+      return ratios.every( (val, _i, arr) => val === arr[0] );
```

Comment:
We can't use arrow syntax, because that's only in ES6.
This comparison shouldn't be strict equality, but `math.isclose` again.